Week_04/Lecture_8/assignment3/src/main/java/org/assignment1/assignment3/service/employeeService.java

```diff
@@ -0,0 +1,66 @@
+package org.assignment1.assignment3.service;
+
+import org.assignment1.assignment3.model.employee;
+import org.assignment1.assignment3.repository.primaryEmployeeRepository;
+import org.assignment1.assignment3.repository.secondaryEmployeeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@Transactional
```

Comment:
Why we put @Transction in 2 place (class and method)?

Reply:
It's a mistake (previously I try to use Transactional on Class but forget to delete when I use in the method) and I already fix this in new commit, Thank you Ms for the feedback.